process/project_milestones.md

```diff
@@ -0,0 +1,51 @@
+
+## Sandbox
+
+Some milestones may not apply to all projects and should be leveraged as guiding points. Milestones are described as the desired state with examples for how to achieve this.
+* Project has the basics of security underway
+  * Completion of the [self-assessment](https://github.com/cncf/tag-security/blob/main/assessments/guide/self-assessment.md)
+* Project holds community meetings or active discussions
+  * Zoom with meeting notes occurring more than twice a quarter (or community.cncf.io!) 
+  * Use of GitHub discussions, issues, or a group messaging service (slack, wechat, etc.)
+* Project guides community contributions
+  * PRs from community members receive timely feedback and discussion from the maintainers
+* Project works with an angel adopter
+  * Partners closely with the project to address organizational needs of the project early on
+* Project begins solidifying a versioning schema and release cadence
+  * A roadmap clearly defines initiatives that align with releases
+  * Releases begin to have establish a pattern of regularity
+* Project has engaged a Technical Advisory Group for feedback on some portion of the project relevant to the domain
+* Project has a clearly discoverable governance doc that covers the basics of decision making to include the commit bit
+* TBD - add more!
+
+## Incubation
+
+Incubation milestones are guiding points to highlight success areas a maturing project can accomplish before Graduation. Not all Incubating projects will achieve every milestone, however many successful graduated projects have demonstrated these milestones prior to their Graduation. How each project achieves some or all of these milestones will vary widely. Projects do not need to meet this in order to graduate, rather these assist projects in meeting the Graduation criteria.
+* Project has a robust set of governance documentation that defines how the direction of the project is decided upon and managed which considers company diversity in decision making
+* Project can demonstrate its application, practice, and adjustments to its governance documentation as a result of regular project operations
+* Project has a robust and mature security posture as evidenced by
+  * Threat model
+  * [Joint review](https://github.com/cncf/tag-security/tree/main/assessments#components-of-the-security-review-package)
+  * First passing then Silver OpenSSF Best Practice Badge 
```

Comment:
This is currently different than graduation requirements. I would keep it consistent across both places https://github.com/cncf/toc/blob/main/process/graduation_criteria.md#graduation-stage

Reply:
@xmulligan it is different from graduation requirements. Perhaps this is better suited as a split? Achieve Passing during incubation (to align with graduation) and then Silver post graduation?  these are not requirements and are optional for project (except where they exist as requirements). Silver has (IMO) reasonable and (in all likelihood)  a fair amount of requirements beyond passing that are already met by highly mature projects.

What do you think of this change? breaking it out and moving Silver to after graduation?

Reply:
As a maintainer, I think this makes sense to still have "goals"/"points for improvement" after graduating as that is not the end of the governance story for a project